src/internal/modules/charge-information/lib/actions.js

```diff
@@ -66,9 +69,9 @@ const setAbstractionData = (request, formValues) => {
 };
 
 const getNewChargeElementData = (request, formValues) => {
-  const { defaultCharges } = request.pre;
+  const { defaultCharges, draftChargeInformation } = request.pre;
   const { step } = request.params;
-  return mappers[step] ? mappers[step](formValues, defaultCharges) : omit(formValues, 'csrf_token');
+  return mappers[step] && draftChargeInformation.scheme !== 'sroc' ? mappers[step](formValues, defaultCharges) : omit(formValues, 'csrf_token');
```

Comment:
What's happening here? Why do we omit `csrf_token`? Could you kindly add a comment for future reference?

Reply:
done